features/support/env.rb

```diff
@@ -56,6 +56,10 @@
   create(:corporate_partner)
 end
 
+Before('@allow_rescue') do
+  ActionController::Base.allow_rescue = true
+end
+
```

Comment:
Why this line is here? This line means in all tests we allow exceptions. Is there what is intended to do it?

Reply:
@tomas-stefano this has been put there so that exceptions are treated in the same way as it will be treated on production. Otherwise it will dump a stack trace of the error rather than redirecting it to the ErrorController to ensure that the 404 page gets picked up.

Which is what we needed.

Reply:
This will allow exceptions to be rescued just for tests that are tagged with `@allow_rescue` 
https://github.com/moneyadviceservice/frontend/pull/1809/files#diff-e76069191d84db196d4a91586c5b8ad9R19